specification/resources/vpc_peerings/vpc_peerings_get.yml

```
@@ -0,0 +1,38 @@
+operationId: vpcPeerings_get
+
+summary: Retrieve an Existing VPC Peering
+
+description: |
+  To show information about an existing VPC Peering, send a GET request to `/v2/vpc_peerings/$VPC_PEERING_ID`.
+
+tags:
+  - VPC Peerings
+
+parameters:
+  - $ref: 'parameters.yml#/vpc_peering_id'
+
+responses:
+  '202':
```

```suggestion
  '200':
```

Looks like the GET request returns 200 OK.